### PR TITLE
index: Add hello nrf cloud OOB firmware to app index

### PR DESCRIPTION
**Pull request by simensrostad**
Add hello nrf cloud OOB firmware to app index

### CODE REVIEW

File: index/hello-nrfcloud.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Hello nRF Cloud",
```
Comment (coderbyheart) [inline index/hello-nrfcloud.json:2]:
```suggestion
    "name": "Thingy:91 X: Hello nRF Cloud firmware",
```

File: index/hello-nrfcloud.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "description": "",
+    "apps": [
+        {
+            "name": "Hello nRF Cloud firmware",
```
Comment (coderbyheart) [inline index/hello-nrfcloud.json:6]:
```suggestion
            "name": "Thingy:91 X: Hello nRF Cloud firmware",
```

File: index/hello-nrfcloud.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            "title": "firmware",
+            "description": "Out of Box firmware for the Thingy:91 X",
```
Comment (coderbyheart) [inline index/hello-nrfcloud.json:8]:
```suggestion
            "description": "This firmware is used with hello.nrfcloud.com",
```

File: index/hello-nrfcloud.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            "kind": "sample",
+            "tags": ["dfu", "lte"],
+            "releases": [
+                {
+                    "date": "latest",
+                    "name": "Latest Release",
+                    "tag": "latest",
+                    "sdk": "latest"
```
Comment (coderbyheart) [inline index/hello-nrfcloud.json:16]:
```suggestion
                    "date": "2024-08-11T14:37:00Z",
                    "name": "v2.0.1",
                    "tag": "v2.0.1",
                    "sdk": "v2.8.0-rc2"
```

Comment (simensrostad) [reply index/hello-nrfcloud.json]: Not a big fan of this. We can't update the app index for every release.

**Review by FilipZajdel — CHANGES_REQUESTED** [review]
Hi, I would like you to also make a change in the generate-index-json.ts file, line 25: add `'hello-nrfcloud'` to the `nordicOrgs` list.

**Review by FilipZajdel — APPROVED** [review]

**Pull request merged**